CLAUDE.md

```
@@ -181,6 +181,20 @@ The import hierarchy must be respected to prevent circular dependencies:
 - Always use translation keys instead of hardcoding text strings
 - Follow the established pattern for translation keys: `namespace__action_or_description`
 
+### Updating Translation Keys
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)